### PR TITLE
Add windows services supported in WINC

### PR DESCRIPTION
**Pull request by mburke5678**
https://issues.redhat.com/browse/OCPBUGS-2292

Preview: [Windows node services](https://51617--docspreview.netlify.app/openshift-enterprise/latest/windows_containers/understanding-windows-container-workloads.html#windows-node-services_understanding-windows-container-workloads)

### CODE REVIEW

File: modules/windows-node-services.adoc [hunk 1]
```diff
@@ -22,6 +22,12 @@ The following Windows-specific services are installed on each Windows node:
 |Windows Machine Config Bootstrapper (WMCB)
 |Configures the kubelet and CNI plug-ins.
 
+|Windows Explorer
+|Provides a graphical user interface for accessing the file systems.
```
Comment (aravindhp) [inline modules/windows-node-services.adoc:26]: This is `windows_exporter which is a Prometheus exporter for Windows nodes.`

**Review by aravindhp — CHANGES_REQUESTED** [review]

File: modules/windows-node-services.adoc (hunk 1 above)
Comment (aravindhp) [reply modules/windows-node-services.adoc]: https://github.com/openshift/prometheus-community-windows_exporter

Comment (mburke5678) [reply modules/windows-node-services.adoc:26]: Sorry! I can't read apparently!

**Review by aravindhp — COMMENTED** [review]
/lgtm

**Comment by jfrancoa** [comment]
The name of the commit is a little confusing, it says "not supported in WINC", however, those are services installed on each of the Windows worker nodes, so they are indeed supported. Did I miss anything?

**Comment by aravindhp** [comment]
Good catch @jfrancoa. I completely missed that. Yes, we are listing all the services running on a Windows node here.

**Comment by mburke5678** [comment]
I fixed the title of the PR. Apologies. Trying to tackle too many issues at once and am confusing myself and others.

**Comment by bscott-rh** [comment]
LGTM.

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Pull request merged**